tensorflow/contrib/go/README.md

```diff
@@ -0,0 +1,163 @@
+# TensorFlow for Go
+
+This package provides a high-level Go API for TensorFlow, this package
+provides the necessary tools to create and manipulate Tensors, Variables,
+Constants as also to build, load and run Graphs.
+
+## API documentation
+* [Session](g3doc/session.md): Encapsulates the environment in which Operation
+  objects are executed, and Tensor objects are evaluated.
+* [Graph](g3doc/graph.md): Contains a set of Operations, which represent units
+  of computation; and Tensors, which represent the units of data that flow
+  between operations.
+* [Tensor](g3doc/tensor.md): Typed multi-dimensional array.
+
+## Practical Examples
+
+##### Python Graph generated and executed on Go
+
+This is just an example that shows how to interact with the provided API
+
+In order to generate a valid Graph you can use the next Python code
+
+```python
+import tensorflow as tf
+
+input1 = tf.placeholder(tf.int64, shape=(2, 2, 2), name='input1')
+input2 = tf.placeholder(tf.int64, shape=(2, 2, 2), name='input2')
+output = tf.add(input1, input2, name='output')
+
+with tf.Session() as sess:
+    tf.train.write_graph(sess.graph_def, '/tmp/graph/', 'test_graph.pb', as_text=True)
+```
+
+The previous code prepares two placeholders with names 'input1' and
+'input2' respectively and other tensor used as output of the addition of the
```

Comment:
'respectively and other' -> respectively, and another tensor used as the output ...